lib/seccomp/CSystemCallFilter_Windows.cc

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+#include "seccomp/CSystemCallFilter.h"
+
+#include <core/CLogger.h>
+#include <core/CWindowsError.h>
+
+#include <WinBase.h>
+
+
+namespace ml {
+namespace seccomp {
+
+CSystemCallFilter::CSystemCallFilter() {
+    HANDLE job = CreateJobObject(nullptr, nullptr);
+    if (job == nullptr) {
+        LOG_ERROR(<< "Failed to create Job Object: " << ml::core::CWindowsError().errorString());
+    }
+
+    JOBOBJECT_BASIC_LIMIT_INFORMATION limits;
+
+    if (QueryInformationJobObject(job, JobObjectBasicLimitInformation,
+                                  &limits, sizeof(limits), nullptr) == 0) {
+        LOG_ERROR(<< "Error querying Job Object information: " << ml::core::CWindowsError().errorString());
+        return;
```

Comment:
If we return here then it would be best to call `CloseHandle()` on the handle to avoid leaking it.  Same for the two error returns below.